config/routes.rb

```diff
@@ -16,6 +16,13 @@
     resources :manage_users, only: %i[new create]
   end
 
+  scope module: 'teams' do
```

Comment:
After looking into the current routes and controllers a bit, I think what we actually want to do here is add the users resource to the teams resource above, in the same nesting where the 'manage_users' resource is located. This block is essentially the same. I can explain more if you'd like!

Reply:
I think I understand .. I'll take a stab at it now!

Reply:
Never mind, after realizing that we'd have to explicitly point to the teams/users_controller if we changed it as I suggested above, I understand why this is preferable. The scope module: 'teams' line is what's keeping us from explicitly pointing to that controller.